vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogbundleinitialization.rb

```diff
@@ -29,29 +45,41 @@ def get_options_hash(dialog_options)
     option_key = k.downcase.to_sym
     if options_regex =~ k
       sequence_id = Regexp.last_match[2].to_i
+      next if v.blank?
 
-      unless v.blank?
-        $evm.log("info", "Adding sequence_id:<#{sequence_id}> option_key:<#{option_key.inspect}> v:<#{v.inspect}> to options_hash")
-        if options_hash.key?(sequence_id)
-          options_hash[sequence_id][option_key] = v
-        else
-          options_hash[sequence_id] = {option_key => v}
+      $evm.log("info", "Adding seq_id:<#{sequence_id}> key:<#{option_key}> value:<#{v}>")
+      if options_hash.key?(sequence_id)
+        options_hash[sequence_id][option_key] = v
+      else
+        options_hash[sequence_id] = {option_key => v}
```

Comment:
Both of these conditions are handled by store_path, so this all simplifies to...

```ruby
options_hash.store_path(sequence_id, option_key, v)
```